doc/Makefile

```diff
@@ -43,44 +44,49 @@ help:
 	@echo "  doctest    to run all doctests embedded in the documentation (if enabled)"
 	@echo "  updaterst  to update sphinx rst to reflect code structure changes"
 
+schema.md: parse_gx_xsd.py schema_template.md ../lib/galaxy/tools/xsd/galaxy.xsd ## Build Github-flavored Markdown from Galaxy Tool XSD (expects libxml in environment)
+	python parse_gx_xsd.py schema_template.md ../lib/galaxy/tools/xsd/galaxy.xsd > $@
```

Comment:
It is called only one time, the first line should be read as: `schema.md` should be regenerated if any of the prerequisites (`parse_gx_xsd.py`, `schema_template.md`, `../lib/galaxy/tools/xsd/galaxy.xsd`) has changed after this file.

`$@` is a special `make` variable which is substituted with the make target, as you were correctly supposing.